package-lock.json

```diff
@@ -23342,4 +23342,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
I guess this file was missing its final line break. I suppose fixing this is a good thing.